Ginger/GingerCoreNET/Run/RunListenerLib/LiteDBRepository.cs

```diff
@@ -212,6 +217,39 @@ public override object SetReportBusinessFlow(Context context, bool offlineMode,
             context.Runner.CalculateBusinessFlowFinalStatus(context.BusinessFlow);
             BFR.SetReportData(GetBFReportData(context.BusinessFlow, context.Environment));
             BFR.Seq = ++this.bfSeq;
+
+            int ChildExecutableItemsCountActivity = 0;
+            int ChildExecutedItemsCountActivity = 0;
+            int ChildPassedItemsCountActivity = 0;
+
+            ChildExecutableItemsCountActivity = context.BusinessFlow.Activities.Count;
```

Comment:
notice that Activity.Actions stats calc and BusinessFlow.Activities stats calc is exactly the same, you can create generic function which will do the calc and return you List of results

Reply:
ignore